build.sbt

```diff
@@ -633,7 +634,7 @@ lazy val refined: ProjectMatrix = (projectMatrix in file("integrations/refined")
   )
   .jvmPlatform(scalaVersions = scala2And3Versions)
   .jsPlatform(
-    scalaVersions = scala2And3Versions,
+    scalaVersions = List(scala3),
```

Comment:
@majk-p What was the reason for this change? tapire-refined is now not available for Scala 2 / Scala.js anymore.

Reply:
I intended to apply this to iron since it's only available for Scala 3. My bad, sorry, let me issue a fix.

Reply:
https://github.com/softwaremill/tapir/pull/3129 @adamw @kciesielski would you mind having a look? sorry for the mess :sweat_smile: